index.js

```diff
@@ -53,6 +55,11 @@ module.exports = {
       return defaultConfig;
     }
 
+    // if getting parent file name from module failed then use error-stack to fetch the parent file path
```

Comment:
Please add a detailed explanation in the PR description. It's very hard to understand the whole context around this.

When the bug occurs `module.parent.filename` contains the first file that called switch right ? So in failure cases first if will never return defaultConfig, therefore we added a second one.


Why ? 
What ? 
How?